target/linux/ramips/dts/mt7621_dlink_dir-882-r1.dts

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
drop dts-v1, it's in mt7621.dtsi now ...

Reply:
Done.